### PR TITLE
Upgrade 1

### PR DESCRIPTION
**Pull request by zmanian**
To validate this export,

Checkout this bug fix branch
https://github.com/cosmos/cosmos-sdk/pull/3148

Sync a node with pruning disabled to current `game_of_stakes`.

`gaiad start --pruning=nothing`
`gaiad export --height 1300 --for-zero-height`

Verify against the new genesis. The only expected change is in `max_bytes` and `chain-id`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,26 @@ chat here: [Riot](https://matrix.to/#/!RKBbCjMEiDPKKewRIE:matrix.org?via=matrix.
 
 ## Updates
 
+### Upgrade 1.
+
+We chose a very conservative initial blocksize of `50kb` for the intitial game of stakes launch as a conservative starting.
+
+With 199 validatiors online, the signatures in blockheader now take up most of the block.
```
Comment (cwgoes) [inline README.md]:
```suggestion
 With 199 validators online, the signatures in each block header now take up most of the block.
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,26 @@ chat here: [Riot](https://matrix.to/#/!RKBbCjMEiDPKKewRIE:matrix.org?via=matrix.
 
 ## Updates
 
+### Upgrade 1.
+
+We chose a very conservative initial blocksize of `50kb` for the intitial game of stakes launch as a conservative starting.
```
Comment (cwgoes) [inline README.md]:
```suggestion
 We chose a very conservative initial blocksize of `50kb` for the intitial game of stakes launch as a conservative starting value.
```

File: README.md [hunk 3]
```diff
@@ -19,6 +19,26 @@ chat here: [Riot](https://matrix.to/#/!RKBbCjMEiDPKKewRIE:matrix.org?via=matrix.
...
+ With 199 validators online, the signatures in each block header now take up most of the block.
+
+We will be doing the first network upgrade.
+
+To upgrade, download the new genesis to your genesis folder.
+
+`gaiad unsafe-reset-all`
+
+`gaiad start` with command line flags as appriopiate for your node type.
+
```
Comment (alexanderbez) [inline README.md]: Can we remove all these newlines?

**Comment by Slamper** [comment]
Can confirm that the genesis file is valid with a shasum of `907e0e62b50382bd735adb413afff75c45d6d1a6`

**Comment by adrianbrink** [comment]
Could someone provide a summary of the proposed change? It is impossible for someone to look at this proposal in isolation (without context from Riot) and understand what it is proposing.

**Comment by zmanian** [comment]
This would need to be re-exported if we were to move forward with it.

**Comment by Slamper** [comment]
The governance proposal `3` on `game_of_stakes_3` was rejected so it looks like we can close this.

**Pull request closed without merging**